net/frr/src/etc/rc.syshook.d/start/50-frr

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
```

Comment:
it's ok to try and ditch this, but it's also hard to predict that this isn't needed anymore because it handles a startup error edge case. Functionally this shouldn't matter. If we are worried about the error being shown a `|| true `would do the trick, or checking if already running before starting. But only commenting here for future us. ;)